src/app/_components/PortalComponents/Portal/recruitment/columns.tsx

```
@@ -57,8 +57,8 @@ export const columns = [
         newForm={false}
       />
     ),
-    enableHiding: false,
     id: "edit",
+    enableHiding: false,
```

ESLint

[error] Expected object keys to be in ascending order. 'enableHiding' should be before 'id'.

(sort-keys)